doc/Extensions/Distributed-Poller.md

```diff
@@ -56,9 +56,25 @@ Central storage should be provided so all RRD files can be read from and written
 
 For this example, we are running RRDCached to allow all pollers and web/api servers to read/write to the rrd files with the rrd directory also exported by NFS for simple access and maintenance.
 
+Install RRD tool from source on the pollers and your main system to make sure you have RRDTool >=1.5.5. RRDTool 1.7 is the lastest version.
```

Comment:
I'm not sure we should have this. We don't care where users install rrdtool from and as time goes on distros will have newer versions meaning this doc will become out of date.